CHANGELOG.asciidoc

```diff
@@ -37,6 +37,7 @@ The latter uses https://github.com/elastic/ecs-logging-java[ecs-logging-java] to
 
 [float]
 ===== Bug fixes
+* Ingres/Actian connection string parsing throws `NumberFormatException` - {pull}1198[#1198]
```

Comment:
[minor] Personal preference, but I prefer describing that we fixed a bug vs describe the bug

```suggestion
* Fix `NumberFormatException` when parsing Ingres/Actian JDBC connection strings - {pull}1198[#1198]
```